site/content/docs/5.2/forms/form-control.md

```diff
@@ -52,6 +52,14 @@ Add the `readonly` boolean attribute on an input to prevent modification of the
 
 If you want to have `<input readonly>` elements in your form styled as plain text, replace `.form-control` with `.form-control-plaintext` to remove the default form field styling and preserve the correct `margin` and `padding`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant with a **readonly plain text** input element and with **labels** that are **not above input fields**, should not be used because it does no respect the Orange Design System specifications.
```

Comment:
Maybe simplify the message by just saying that the example is valid conceptually but that it is only the horizontal layout that is a problem.

Reply:
In fact, it's not only the horizontal layout that is the problem, but also the readonly plain text `email@example.com`. In addition, we used the "with **labels** that are **not above input fields**" in severals other callout messages.

Proposition of slight variation:
> This form variant, with a **readonly plain text** input element and **labels** that are **not above the input fields**, should not be used because it does no respect the Orange Design System specifications.

Reply:
Seems to be fine as-is ?